### ISSUE TITLE
Font.new not handling relative paths

### ISSUE THREAD

**Issue by ericathegreat** [gosu/gosu#380]
Passing through a path using the :name option only seems to work when the path is absolute. ie:
Gosu::Font.new 20, name: 'C:/Users/Erica/Documents/marco7/lib/marco/ui/Xolonium-Regular.ttf'

Running the code with a relative path:
Gosu::Font.new 20, name: './Xolonium-Regular.ttf'
produces the following error:
```
ERROR:  failed to open './Xolonium-Regular.ttf'
ERROR: CreateFile failed, GetLastError() = 0x2
```

This is happening using Ruby 2.2.1 on Windows 7, with Gosu 0.11.2.

**Comment by ericathegreat** [comment]
Should also note: there is a relatively easy work around. Just add #{__dir__} at the start of the string. Not immediately obvious though, given everything else works relative.

**Comment by jlnr** [comment]
Ah, thanks for reporting this. This error only affects Windows, which uses a bit of awkward code in `TextTTFWin.cpp` to extract the font name of a TTF file. One more reason to switch to stb_ttf, although this particular error should be easy to fix.

**Comment by jlnr** [comment]
Fixed - I'll try to release this soon along with the font fix from #384.
